ecosystem/sep-0030.md

```diff
@@ -14,7 +14,7 @@ Version: 0.6.1
 
 ## Summary
 
-This protocol defines an API that enables an individual (e.g., a user or wallet) to regain access to a Stellar account that it owns after the individual has lost its private key without providing any third party control of the account. Using this protocol, the user or wallet will preregister the account and a phone number or email with two or more servers implementing the protocol and add those servers as signers of the account. No individual server will have control of the account, but collectively, they may help the individual recover access to the account. The protocol also enables individuals to pass control of a Stellar account to another individual.
+This protocol defines an API that enables an individual (e.g., a user or wallet) to regain access to a Stellar account that it owns after the individual has lost its private key without providing any third party control of the account. Using this protocol, the user or wallet will preregister the account and a phone number or email or other form of authentication with one or more servers implementing the protocol and add those servers as signers of the account. If two or more servers are used no individual server will have control of the account, but collectively, they may help the individual recover access to the account. The protocol also enables individuals to pass control of a Stellar account to another individual.
```

Comment:
I think this statement: `If two or more servers are used no individual server will have control of the account` seems inaccurate, or at least incomplete. It is true only if the account that is being registered is correctly configured, as you later describe in this PR.

So maybe we could change the wording to say `If two ore more servers are used and the account has the right signer configuration, no individual server will have control of the account`. What do you think?

Reply:
That's a great point. Actually that raises another point, you can use a single server and still not give the server control because you could use a SEP-5 mnemonic phrase as the other party to a multi-party recovery. I think I need to revisit the phrasing here and in the paragraphs further down too, so that it is less opinionated but not so general as to be meaningless.

Reply:
Made this change in 64e8382.